rocket/src/main/scala/craft/ShiftRegisterMem.scala

```diff
@@ -17,8 +17,7 @@ object ShiftRegisterMem {
     if (n == 0) {
       in
     } else if (n == 1) {
-      val out = Reg(in.cloneType)
-      out := in
+      val out = RegEnable(in, en)
```

Comment:
This is basically ok but I would prefer to see the parameters labelled when variable names don't provide
usage context